src/Tribe/Views/V2/Messages.php

```diff
@@ -94,6 +94,10 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found.',
 				'the-events-calendar'
 			),
+			'no_upcoming_events'               => __(
+				'There are no upcoming events.',
```

Comment:
Can we use the function for the word "events"? At least where we change the code 😉